_data/projects/mutabilitydetector.yml

```diff
@@ -0,0 +1,11 @@
+name: Mutability Detector
+desc: Lightweight analysis tool for detecting mutability in Java classes
+site: mutabilitydetector.org
+tags:
+- java
+- immutability
+- testing
+- static analysis
+upforgrabs:
+  name: up-for-grabs
```

Comment:
I see you have a `newcomer` tag on the site - feel free to use that here if you think it fits better for your project over our `up-for-grabs` tag...